python/device/device.py

```diff
@@ -78,16 +78,47 @@ class DeviceRunConfig(object):
     Currently hardcoded. Later, this will come from desired properties.
     """
 
+    # All durations are in seconds
     def __init__(self):
-        self.max_run_duration = 90
+        # how long should teh test run before finishing.  0 = forever
```

Comment:
misspelled 'the'

Reply:
huh?

---
In reply to: [512842653](https://github.com/Azure/iot-sdk-longhaul/pull/17#discussion_r512842653) [](ancestors = 512842653)

Reply:
the not teh

Reply:
fixed


---
In reply to: [512867206](https://github.com/Azure/iot-sdk-longhaul/pull/17#discussion_r512867206) [](ancestors = 512867206,512842653)